docs/guides/usage/tables.livemd

```diff
@@ -2746,12 +2746,12 @@ This is what the "mapping" aspect of match specs refers to: that we can not only
 
 <!-- livebook:{"break_markdown":true} -->
 
-Just as data in a list in Elixir need not be homogenous, but can contain any term; objects in the same `:ets` table can be any term — just so long as they are all tuples, and all have a key entry at the same position in the tuples:
+Just as data in a list in Elixir need not be homogeneous, but can contain any term; objects in the same `:ets` table can be any term — just so long as they are all tuples, and all have a key entry at the same position in the tuples:
 
 ```elixir
-hetrogenous_table = :ets.new(HetrogenousTable, [:set])
+heterogenous_table = :ets.new(heterogenousTable, [:set])
```

Comment:
The spell checker changed the case of "HeterogenousTable" in the function call.  The first parameter to this call is an atom hence it needs to start with an upper-case "H". This needs to be fixed.